enacts/calc.py

```diff
@@ -746,6 +746,271 @@ def daily_to_dekad(daily_data, method=None, method_kwargs={}, time_dim="T"):
     return dekad_data
 
 
+def resample_interval_to_daily(time_series, extent=None, time_dim="T_bins"):
+    """ Resample any (interval-based) time series to daily
+
+    Parameters
+    ----------
+    time_series : xr.DataArray or xr.Dataset
+        data depending on time intervals greater or equal then a day
+    extent : str, optional
```

Comment:
Make this a boolean instead of a string, and call it `is_extensive` (or `is_intensive`, your choice).

Also, maybe it's just me but I would be helped by definitions or examples (temperature, rainfall). I can never remember which is which.